apps/web/components/getting-started/components/ConferenceAppItem.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from "react";
+
+import useAddAppMutation from "@calcom/app-store/_utils/useAddAppMutation";
+import { InstallAppButton } from "@calcom/app-store/components";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { trpc } from "@calcom/trpc/react";
+import type { App } from "@calcom/types/App";
+import { Button, showToast } from "@calcom/ui";
+
+interface IConferenceAppItem {
+  title: string;
+  description?: string;
+  imageSrc: string;
+  type: App["type"];
+  variant: App["variant"];
+  slug: App["slug"];
+  installed: App["installed"];
+  credentialIds: number[];
+}
+
+const ConferenceAppItem = (props: IConferenceAppItem) => {
```

Comment:
you returned it back or why  AppItem is created ?

Reply:
AppItem represents each items in this list
<img width="678" alt="image" src="https://user-images.githubusercontent.com/110449633/230605636-b778d27b-10fc-4587-9f44-752bd678de0f.png">

Reply:
+ for Zoom integration we still have to keep this `completedOnboarding` in session or add it to this file [InstallAppButton](https://github.com/join-com/cal-com/blob/f52afd6602c69dab263b2f7f7ececc363dfd360c/packages/app-store/zoomvideo/components/InstallAppButton.tsx#L6) + update Zoom callback file